GenumKit/Stencil/GenumTemplate.swift

```diff
@@ -7,42 +7,41 @@
 import Stencil
 
 // Workaround until Stencil fixes https://github.com/kylef/Stencil/issues/22
-public class GenumTemplate: Template {
+open class GenumTemplate: Template {
   public override init(templateString: String) {
     let templateStringWithMarkedNewlines = templateString
-      .stringByReplacingOccurrencesOfString("\n\n", withString: "\n\u{000b}\n")
-      .stringByReplacingOccurrencesOfString("\n\n", withString: "\n\u{000b}\n")
+        .replacingOccurrences(of: "\n\n", with: "\n\u{000b}\n")
```

Comment:
Not sure why I did the replacement twice, but I think that was intentional (like for cases like `\n\n\n`, the first replacement only replaces the first 2 `\n` and the second takes care of the remaining ones)… so we should keep that dual replacement in place even after the Swift 3 migration.